src/main/java/org/infoshareacademy/javaholics/route/RouteService.java

```diff
@@ -1,4 +1,149 @@
 package org.infoshareacademy.javaholics.route;
 
+import java.util.Date;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
 public class RouteService {
+    boolean status;
+    Date date = new Date();
+    Route route = new Route();
+
+    public Date getDate() {
+        return date;
+    }
+
+    Scanner scanner = new Scanner(System.in);
+
+    public void routeLength() {
+        boolean error = false;
+        System.out.println("Podaj początek trasy : ");
+        route.setPlaceStart(scanner.nextLine());
```

Comment:
warto zadbać, aby sprawdzało, czy to na pewno nazwa. Wpisałem numer i przeszło :)